### ISSUE TITLE
Plugin not working anymore

### ISSUE THREAD

**Issue by GuillaumeLeclerc** [Quramy/tsuquyomi#61]
Hello,

For an unknown reason I can't get completion anymore. The plugin ins installed and the Tsu.. commands are available. but sometimes I get this error:

`HTTPConnectionPool(host='127.0.0.1', port=42529): Read timed out. (read timeout=30)`

Thank you for your help

**Comment by GuillaumeLeclerc** [comment]
Precision: The syntax checker still work (I get the side window with errors) but omni completion (Cx-Co) does not work

**Comment by Quramy** [comment]
Hi @GuillaumeLeclerc .

My plugin relies on tsserver process.

Please confirm whether tsserver's process is working.

In vim, exec `:TsuStatusServer' and, 
If there is 'undefined', tsserver does not start.
If there is 'reading' or 'idle', your tsserver was started.

or in your terminal exec `ps aux | grep tsserver` and check that there is some PID.

**Comment by Quramy** [comment]
Or maybe your issue is related to #47 , please check your typescript version. 
v1.7.3 or earlier sometimes cause timeout in completion.

**Comment by GuillaumeLeclerc** [comment]
I updated typescript from version 1.5.3 to 1.8.9. and it didn't solved the problem.

The server is reading or idle (sometimes stays a very long time in reading state)

As I said the compile errors does shows up. Only auto-complete with C-x X-o does not work. Maybe I installed a plugin that has a conflict in bindings. 

I also noticed that the normal complete C-n does not work any more (I need to do C-x C-n to show it)
